index.html

```diff
@@ -155,46 +159,42 @@ <h3>Meetings</h3>
     </div>
 
     <div id="collaboration">
-      <h2 class="collaboration-h2">Collaboration</h2>
-      <h3>Get involved with our community and our work</h3>
+      <h2 class="collaboration-h2">Get Involved</h2>
       <p class="collaboration-p">
-        The APG Task Force conducts its work using a variety of synchronous and
-        asynchronous tools. And there are several ways to get involved.
+        The APG Task Force relies on broad community representation and participation to continuously improve the usefulness and quality of the APG.
+        There are a variety of ways you can get involved and help promote development of accessible experiences. 
       </p>
       <ul>
         <li>
-          <h3>Join Our Community</h3>
+          <h3>Join the Task Force</h3>
           <p>
-            To join the APG Task Force, individuals must be participants of the
-            ARIA Working Group. Participants are expected to actively contribute
-            to the work of the task force.
+            To join the APG Task Force, individuals need to first join the W3C ARIA Working Group.
+            Participants are expected to actively contribute to the work of the task force.
           </p>
           <a
             rel="noopener noreferrer"
             target="_blank"
             href="https://www.w3.org/WAI/ARIA/task-forces/practices/"
           >
-            Connect With Us
+            Learn more about the work of the task force and how to join
           </a>
           <img
             alt="An icon showing three nodes connecting."
             src="img/index-6.svg"
          >
         </li>
         <li>
-          <h3>Contribute to Our Project</h3>
+          <h3>Contribute via GitHub</h3>
           <p>
-            To contribute without joining the task force, see the ARIA Working
-            Group contribute page for general instructions. To contribute to
-            documents under development, see how to contribute to the source
-            repository directly.
+            Many valuable contributions are made by people who find or raise issues of interest in our GitHub repository and then submit proposed changes via a GitHub pull request.
+            If you choose this path, it is tremendously helpful if you first study the APG's contribution guidelines.
```

Comment:
@mcking65 

The contents look good and the tone is very kind. ;-)
Only one thing is that this sentence can be rewritten. there are two If sentences in one sentence.